webui/react/src/pages/OldProjectDetails.tsx

```diff
@@ -152,6 +152,14 @@ const ProjectDetails: React.FC = () => {
   const { settings, updateSettings, resetSettings, activeSettings } =
     useSettings<ProjectDetailsSettings>(settingsConfig);
 
+  const tableOffset = (() => {
+    if (settings.tableOffset >= total && total) {
```

Comment:
nit: `if (total && settings.tableOffset >= total)` is better cuz of short circuit evaluation